cardstack/header.php

```diff
@@ -1,3 +1,8 @@
+<?php
+require_once( __DIR__ . '/functions.php');
```

Comment:
Is this needed? Check at what stage functions.php is executed in WordPress. The class may already be available in this scope.

Reply:
Done